SpatialGDK/SpatialGDK.uplugin

```diff
@@ -58,7 +58,7 @@
       "Enabled": true
     },
     {
-      "Name": "GameplayAbilities",
+      "Name": "ReplicationGraph",
```

Comment:
Is it on purpose, that `GameplayAbilities` is removed here?

Reply:
Yep - should have mentioned, this is a driveby - it was a no-longer required dependency.